src/js/common/utils/helpers.js

```diff
@@ -153,10 +153,12 @@ export function scrollToFirstError() {
 }
 
 export function scrollAndFocus(errorEl) {
-  const currentPosition = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0;
-  const position = errorEl.getBoundingClientRect().top + currentPosition;
-  Scroll.animateScroll.scrollTo(position - 10, getScrollOptions());
-  focusElement(errorEl);
+  if (errorEl) {
```

Comment:
Should we be using refs to obtain this instead? If so, we could add this to the refactor ticket.

Reply:
Yeah, we should probably use refs for the error message in PhotoField. I'll add it to the refactor ticket.